apps/zentao/21.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   zentao:
-    image: hub.zentao.net/app/zentao:21.2
+    image: hub.zentao.net/app/zentao:21.3
     container_name: ${CONTAINER_NAME}
     restart: always
     ports:
```

Comment:
There is one change noted in your changeset:

- The Docker image tag was updated from `hub.zentao.net/app/zentao:21.2` to `hub.zentao.net/app/zentao:21.3`.

This is an update to reflect the version of Zento that the container should be running. If you need further assistance with this code snippet or have additional questions about maintaining and optimizing Docker configurations, feel free to ask!